boards/components/src/gpio.rs

```diff
@@ -51,8 +51,8 @@ use kernel::static_init_half;
 #[macro_export]
 macro_rules! gpio_component_helper_max_pin {
     () => { 0usize };
-    ($a: expr, $b: expr, $($tail:expr,)*) => { $crate::gpio_component_helper_max_pin! (max ($a, $b), $($tail,)*) };
-    ($a: expr,) => { $a };
+    ($a:expr, $b:expr, $($tail:expr),* $(,)?) => { $crate::gpio_component_helper_max_pin! (max ($a, $b), $($tail,)*) };
+    ($a:expr $(,)?) => { $a };
```

Comment:
Quite interestingly, here a trailing comma was actually _required_ in the arguments. :)